src/main/java/com/blog/stream/pagination/PaginationUtils.java

```diff
@@ -1,63 +0,0 @@
-package com.blog.stream.pagination;
-
-import java.util.function.Supplier;
-import java.util.stream.Stream;
-import java.util.stream.StreamSupport;
-
-import static com.blog.stream.pagination.PageSpliterator.PAGED_SPLITERATOR_CHARACTERISTICS;
-
-public final class PaginationUtils {
```

Comment:
The PageSpliterator is intended to be an internal implementation thing, so the PaginationUtils would potentially be the only thing visible in the library. 

I.e. the client of this lib doesnt need to even know the word spliterator (I didnt until I message around with this 😄 )

Let me know what you think